tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/CoreMessagingProxy.java

```diff
@@ -30,21 +33,20 @@
 
    private final String resourceName;
 
-   private final ClientSession session;
+   private final ServerLocator locator;
 
-   private final ClientRequestor requestor;
+   private ClientSessionFactory sessionFactory;
```

Comment:
If you are always opening / closing on each call, why not to make it part of the method.

What was the reason to not reuse the session?